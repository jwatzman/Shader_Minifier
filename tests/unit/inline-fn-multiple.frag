uniform vec3 params[42];

vec3 i_getFoo() {
  return params[4];
}

vec3 f1() {
  return i_getFoo();
}

vec3 f2(float a) {
  return i_getFoo() + a;
}
