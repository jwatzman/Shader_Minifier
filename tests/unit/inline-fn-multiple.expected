uniform vec3 v[42];
vec3 N()
{
  return v[4];
}
vec3 N(float N)
{
  return v[4]+N;
}
